### PR TITLE
cli: add `--cs.create_empty_blocks_interval` flag

### PR DESCRIPTION
**Pull request by jgimeno**
Closes https://github.com/tendermint/tendermint/issues/4065

* [x ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [x ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ program](https://hackerone.com/tendermint).
 - [types] \#2521 Add `NumTxs` to `BlockMeta` and `EventDataNewBlockHeader`
 - [docs] [\#4111](https://github.com/tendermint/tendermint/issues/4111) Replaced dead whitepaper link in README.md
 - [p2p] [\#4185](https://github.com/tendermint/tendermint/pull/4185) Simplify `SecretConnection` handshake with merlin
+- [cli] [\#4065](https://github.com/tendermint/tendermint/issues/4065) Add flag to set create_empty_blocks_interval programmatically.
```
Comment (melekes) [inline CHANGELOG_PENDING.md]:
```suggestion
- [cli] [\#4065](https://github.com/tendermint/tendermint/issues/4065) Add `--consensus.create_empty_blocks_interval` flag (@jgimeno)
```

**Comment by jgimeno** [comment]
@melekes remember to update this page, https://docs.tendermint.com/master/tendermint-core/using-tendermint.html

So it appears that it can be set by command line too the interval. Thanks!!

**Comment by melekes** [comment]
@jgimeno want to do it?

**Comment by jgimeno** [comment]
Oh yeah, did not see that it was possible from the repo!

**Pull request merged**